src/main/java/org/fundacionjala/sfdc/pageobjects/Contacts/SFCNewModifyPage.java

```diff
@@ -0,0 +1,88 @@
+package org.fundacionjala.sfdc.pageobjects.Contacts;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.Alert;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.Alert'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)